exporter/opentelemetry-exporter-jaeger/src/opentelemetry/exporter/jaeger/__init__.py

```diff
@@ -120,6 +121,7 @@ class JaegerSpanExporter(SpanExporter):
         insecure: True if collector has no encryption or authentication
         credentials: Credentials for server authentication.
         transport_format: Transport format for exporting spans to collector.
+        udp_split_oversized_batches: Re-emit oversized batches in smaller chunks.
```

Comment:
Why only udp? Why not simply provide a size? But I am curios how this size is calculated.

Reply:
Because in the case of TCP your stack will just fragment your message.